src/hash.h

```diff
@@ -132,15 +132,17 @@ class CHashWriter
 {
 private:
     CHash256 ctx;
+    size_t nBytesHashed;
```

Comment:
The byte counts are not used in the rules.  @gandrewstone just included them for reporting.